scripts/setup_influx.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
```

Comment:
Can you change this to
```
#!/bin/sh
```

It doesn't look like we have use any bash-specific syntax, so might as well avoid the dependency.